<launch>

	<node pkg="px4ctrl" type="test_fusion" name="test_fusion" output="screen">
		
		<remap from="~odom" to="/vicon/odom" />
		
		<remap from="~acc" to="/vrpn_client_node/Quad13_ipn/accel" />
		
		<remap from="~cmd" to="/position_cmd" />

        <rosparam command="load" file="$(find px4ctrl)/config/fusion.yaml" />
	</node>
 
</launch>
